src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -1272,3 +1275,105 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_bounded_relu()
     auto m = std::make_shared<pattern::Matcher>(min, callback);
     this->add_matcher(m);
 }
+
+void ngraph::runtime::cpu::pass::CPUFusion::construct_horizontal_fusion()
```

Comment:
@jbobba could you please suggest a more descriptive name? this is your forté 😄

Reply:
i'm ok with horizontal_fusion, may be construct_conv_horizontal_fusion